lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -109,4 +115,30 @@ sub press_accept_button {
     send_key('alt-a');
 }
 
+sub open_clone_partition_dialog {
+    my ($self) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    send_key($self->{partition_table_shortcut});
+    for (1 .. 2) { wait_screen_change { send_key "down" } }
+    send_key "ret";
```

Comment:
I could not use the shortcut for "clone partition" here as wait_screen_change was seeing some screen change where there were none, even after tweaking the match level.

Reply:
yes, that submenu is tricky to access, I remember to have some issues. it is fine like that.